sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java

```diff
@@ -51,7 +54,22 @@
 
     private static final String READONLY_PARTITION_NAME = Constants.READONLY;
 
-    private static final String FAKE_TOKEN = "mock";
+    private static final TokenExchange.TokenExchangeServiceCallback TOKEN_EXCHANGE_SERVICE_CALLBACK =

```

Comment:
static final in Java can either mean static readonly or const in c# terms.
If it means static readonly then the convention is sTokenExchangeServiceCallback (an object like that is not constant compile time expression).